Tests/BootCamp.Chapter.Tests/Lesson3Tests.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.IO;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4852921)